internal/scorecard/config.go

```diff
@@ -66,10 +65,7 @@ func validateConfig(config PluginConfig, idx int, version string) error {
 		if pluginType != "" {
 			return fmt.Errorf("plugin config can only contain one of: basic, olm, external")
 		}
-		pluginType = "external"
-		if schelpers.IsV1alpha2(version) {
-			return fmt.Errorf("revert to v1alpha1 to use external plugins: external plugins are not currently supported with v1alpha2")
-		}
+		return fmt.Errorf("external plugins are not currently supported with v1alpha2")
```

Comment:
We can probably remove all code and mention of external plugins for now (either in this PR or a follow-up)

Reply:
removed plus other external references/tests.